src/test/groovy/com/gradle/ConfigurationCachingFuncTest.groovy

```diff
@@ -0,0 +1,125 @@
+package com.gradle
+
+import org.gradle.testkit.runner.GradleRunner
+import spock.lang.Specification
+import spock.lang.TempDir
+
+import java.nio.file.Paths
+
+class ConfigurationCachingFuncTest extends Specification {
+    static final String CC_PROBLEMS_FOUND = "problems were found storing the configuration cache"
+    static final String PLUGIN_VERSION_SYSTEM_PROPERTY = 'com.gradle.common-custom-user-data-gradle-plugin.version'
```

Comment:
i think we only want this if we're doing cross version testing which I don't think we're doing here.